### PR TITLE
Module 1: Linters installation and Hello Microverse

### PR DESCRIPTION
**Pull request by Yosefgeda**
Installation of linters and 'Hello Microverse' HTML
- Created HTML and CSS files and linked them.
- Then installed Webhint
- After that installed stylelint
- Checked for errors using the installed linters.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,131 @@
...
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
```
Comment (jerryowusu) [inline README.md:35]: The logo is broken. Kindly consider fixing it to make your project look more proffessional :wink:

File: README.md [hunk 2]
```diff
@@ -1 +1,131 @@
...
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank to all Microverse staff.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
```
Comment (jerryowusu) [inline README.md]: Your readme is missing certain key/important sections of the [Microverse Readme template](https://github.com/microverseinc/readme-template) like the `Getting Started`, `Future Features`, `Acknowledgement` etc. Note that it is a requirement to produce all sections of the readme template in this and all other subsequent projects you do in Microverse here. Kindly add them and make sure to customized to match your project credentials. :pray:

**Review by jerryowusu — CHANGES_REQUESTED** [review]
Hi @Yosefgeda , 

Good job so far!
Your project is looking fantastic so far :star_struck: 

However, there are few changes you need to make in order to move on to the next level :sunglasses:

so close

## Highlights
All linters are passing :heavy_check_mark: 
Followed github flow :heavy_check_mark: 

## Required Changes ♻️

- [x] Pull Request should have a descriptive title telling you what changes are inside. Kindly check and update it since your pull request title is not descriptive enough because the whole project is not about linters. :wink: 
- [x] Your pull request does not have any description. Kindly ensure that it your pull request have a  summary describing introduced changes in more detail.
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,131 @@
...
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```
Comment (iambenkis) [inline README.md:31]: - [ ] As requested by the previous reviewer, you should fix the logo link to make your README file more professional,  
What do you think if a recruiter finds a broken link in your project description ? :smile:

File: README.md [hunk 4]
```diff
@@ -1 +1,131 @@
...
+- LinkedIn: [LinkedIn](https://linkedin.com/in/yosefgeda)
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+
+
```
Comment (iambenkis) [inline README.md]: - [ ] Some sections are very important to your README file, like `Getting Started`, `Future Features` etc. My fellow reviewer asked you to add them, happy that you added some of them perfectly but you should make sure you add all of them. :100:

**Review by iambenkis — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @Yosefgeda  👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project, but you are almost there!

## To Highlight 🎉

- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark: 
- PR has a good title and summary. ✔️



 ## Required Changes ♻️



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@iambenkis** in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED 🟢  
### Hi  @Yosefgeda    👋 👋 

#### I'm @TuyishimireEric, Your code reviewer,
Good job so far! 🥇 
done

Your project is complete!  There is nothing else to say other than... it's time to merge it 🦾  
Congratulations! 🎉 

## Highlights
✔️Made the required changes👍 
✔️professional README👍 
✔️No linter👍

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**